src/react/hooks/use-messages.ts

```diff
@@ -1,15 +1,16 @@
 import {
-  DeleteMessageParams,
   Message,
   MessageListener,
   Messages,
   MessageSubscriptionResponse,
   QueryOptions,
   SendMessageParams,
+  UpdateMessageParams,
 } from '@ably/chat';
 import * as Ably from 'ably';
 import { useCallback, useEffect, useState } from 'react';
 
+import { ActionDetails } from '../../../dist/chat/messages.js';
```

Comment:
I think this import is incorrect

Reply:
arhhh I don't know why my editor does this on autocomplete